rpc/src/services/protocol/mod.rs

```diff
@@ -463,53 +463,61 @@ pub(crate) fn proto_get_contract_manager_key(
     }
 }
 
+// TODO: TE-220, be more explicit about the kind of response from the RPC service
+fn handle_rpc_response(response: &ProtocolRpcResponse) -> Result<serde_json::value::Value, failure::Error> {
+    match response {
+        ProtocolRpcResponse::RPCOk(body) => Ok(serde_json::from_str(&body)?),
+        other => Err(failure::err_msg(format!("Got non-OK response from protocol-RPC service: {:?}", other))),
+    }
+}
+
 pub(crate) fn run_operation(chain_param: &str, block_param: &str, json_request: JsonRpcRequest, env: &RpcServiceEnvironment) -> Result<serde_json::value::Value, failure::Error> {
-    let request = create_protocol_json_rpc_request(chain_param, block_param, json_request, FfiRpcService::HelpersRunOperation, &env)?;
+    let request = create_protocol_json_rpc_request(chain_param, block_param, json_request, &env)?;
```

Comment:
@tizoc 
I think we could also allign this name:
`create_protocol_json_rpc_request` -> `create_protocol_rpc_request`
`call_protocol_json_rpc` -> `call_protocol_json_rpc`
like you did in ffi.rs

Reply:
Done, see https://github.com/simplestaking/tezedge/pull/318/commits/f4c7bf8ed5a710c1f2da75c33306c5e0ba532f87 . I also updated the few remaining cases in the OCaml side. Technically everything is still "json", because thats the only thing supported right now, but at least when things change the names will not have to be taken care about too.